runtime/foucoco/src/lib.rs

```diff
@@ -1860,6 +1862,18 @@ impl_runtime_apis! {
 				amount_1_min
 			)
 		}
+
+		fn calculate_remove_liquidity(
```

Comment:
[additional method in zenlink](https://github.com/zenlinkpro/Zenlink-DEX-Module/blob/polkadot-v0.9.38/zenlink-protocol/rpc/src/lib.rs#L71-L77),  applied similar to [interbtc](https://github.com/interlay/interbtc/blob/master/parachain/runtime/interlay/src/lib.rs#L1556-L1567)